### DISCUSSION TITLE
Sleek overall interface scaling? (zoom)

### DISCUSSION THREAD

**Discussion by litew** [ransome1/sleek#238] [General]
Hello,
is there any way to scale/zoom interface in/out? I've tried with Ctrl++ with no luck.

Using sleek with flatpak on Fedora 34.

**Comment by 9z0** [comment]
Click the options icon (second to last in the image below) and select zoom level.

![aa](https://user-images.githubusercontent.com/78097210/128108782-ff59d02a-b43c-437f-8667-a3746514420a.png)

**Reply by litew** [reply]
Awesome, thank you!
